base_country_state_translatable/models/res_country.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# © initOS GmbH 2017
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class CountryState(models.Model):
+    _inherit = 'res.country.state'
+
+    name = fields.Char(translate=True)
```

Comment:
OK, so this one is a simple addon, but we have to maintain it!

May I ask you please to open a pull request to odoo master and propose the change there? I'm pretty sure they'll merge it, and it's [just 1 line of code on their side.
](https://github.com/odoo/odoo/blob/a7fa3cc436517d001717de6b4402c62698b45224/odoo/addons/base/res/res_country.py#L110-L111)

Once you do it, add a note to the roadmap saying that this addon should be dropped in the version where that PR gets merged.